operator/roles/kiali-deploy/tasks/main.yml

```diff
@@ -124,14 +124,21 @@
   - kiali_vars.auth.strategy != 'login'
   - kiali_vars.auth.strategy != 'anonymous'
   - kiali_vars.auth.strategy != 'openshift'
+  - kiali_vars.auth.strategy != 'ldap'
 - name: Confirm auth strategy is valid for Kubernetes environments
   fail:
     msg: "Invalid auth.strategy [{{ kiali_vars.auth.strategy }}]! Must be one of either 'login' or 'anonymous'"
   when:
   - is_k8s == True
   - kiali_vars.auth.strategy != 'login'
   - kiali_vars.auth.strategy != 'anonymous'
-
+  - kiali_vars.auth.strategy != 'ldap'
+- name: Confirm ldap configuration when auth strategy is 'ldap'
+  fail:
+    msg: "Invalid configuration for LDAP! The mandatory parameters should be provided like `ldap_host', 'ldap_port', 'ldap_bind_dn', and 'ldap_base'"
+  when:
+  - kiali_vars.auth.strategy == "ldap"
+  - kiali_vars.auth.ldap.ldap_host == "" or kiali_vars.auth.ldap.ldap_post <= 0 or kiali_vars.auth.ldap.ldap_bind_dn == "" or kiali_vars.auth.ldap.ldap_base == ""
```

Comment:
First bug found from my molecule test. "ldap_post" should be "ldap_port".
@Maneshme I'll fix this in my PR that I wrote to this PR - so when you merge my PR to yours, this will be fixed.

Reply:
@jmazzitelli It seems there are some conflicts in that PR can you check that

Reply:
@Maneshme try it now